backend/config/initializers/devise.rb

```diff
@@ -314,9 +314,16 @@
   # config.sign_in_after_change_password = true
 
   config.jwt do |jwt|
-    jwt.secret = Rails.application.credentials.jwt_secret_key
+    jwt.secret = ENV['JWT_SECRET_KEY'] || Rails.application.credentials.jwt_secret_key
     jwt.dispatch_requests = [['POST', %r{^/login$}]]
     jwt.revocation_requests = [['DELETE', %r{^/logout$}]]
     jwt.expiration_time = 1.year.to_i
   end
+
+  # turn off sessions for user in light of
+  # "Your application has sessions disabled. To write to the session you must first configure a session store"
+  # https://github.com/waiting-for-dev/devise-jwt/issues/235#issuecomment-1475410637
+  config.warden do |warden|
```

Comment:
I kept getting `Your application has sessions disabled. To write to the session you must first configure a session store` when trying to log in and this fixes it but it seems like a hack which I'm not a fan of

Reply:
@mzagaja we're interested in your feedback on this guy in particular